lang/rust/Portfile

```diff
@@ -378,7 +378,7 @@ livecheck.url       https://github.com/rust-lang/rust/tags
 livecheck.regex     refs/tags/(\[\\d\\.\]+).zip
 
 subport rust-src {
-    revision        [ expr ${revision} + 1 ]
```

Comment:
Is this change necessary? I don't really know why the rust-src revision was set to revision + 1...

Reply:
This change doesn't affect rust-src so I was trying to leave its revision untouched. I can either leave the line alone, or nuke it, whatever is simplest.